duckduckgo/iOS
# Content blocker global setting and tutorial

Reviewer: Chris
Asana Tutorial page: https://app.asana.com/0/361428290920652/391317240559345
Global on/off setting: https://app.asana.com/0/361428290920652/407600612467832
Disabled popout: https://app.asana.com/0/361428290920652/407600612467833

**Description**:
Separate content blocker global settings from main panel and show content blocker status in omnibar:
- Move global content blocker toggle to settings
- Highlight when content blocker is disabled in omnibar
- Present separate content blocker panel when disabled to encourage user to reenable
- Add mini tutorial to content blocker panel

**Steps to test this PR**:

**TEST 1**
1. Got to settings and enable the content blocker
2. Browse to a page
3. Ensure that the omnibar shows a site rating
4. Press on the site rating, ensure that the ssl, tracker information and new tutorial appear in the content blocker popout

**TEST 2**
1. Got to settings and disable the content blocker
2. Browse to a page
3. Ensure that the content blocker button is a disabled button (red background with exclamation mark "!")
4. Press the disabled button to show a new panel which allows the content blocker to be reenabled
5. Press the toggle to reenable the content blocker and ensure that the std content blocker panel is displayed

**TEST 3**
1. While browsing, visit the settings page and toggle the content blocker button
2. Ensure the correct button is displayed in the omnibar
3. Now repeat step one (toggling the button again in the other direction) and two

###### Reviewer Checklist:


###### PR DRI Checklist:
